docs/reference/ilm/apis/get-status.asciidoc

```diff
@@ -1,24 +1,24 @@
 [role="xpack"]
 [testenv="basic"]
 [[ilm-get-status]]
-=== Get {ILM} Status API
+=== Get {ilm} status API
```

Comment:
I assume the change to use the long name is intentional here (as well as other header sections) ?